core/commands/fmt.go

```diff
@@ -0,0 +1,33 @@
+package commands
+
+import (
+	"fmt"
+
+	peer "github.com/libp2p/go-libp2p-core/peer"
+	mbase "github.com/multiformats/go-multibase"
+)
+
+func verifyIDFormatLabel(formatLabel string) error {
+	switch formatLabel {
+	case "b58mh":
+		return nil
+	case "b36cid":
+		return nil
+	}
+	return fmt.Errorf("invalid output format option")
+}
+
+func formatID(id peer.ID, formatLabel string) string {
```

Comment:
Since these are only used by the `ipfs key` commands, any reason not to move them to that file? Most of the files in ipfs/core/commands correspond to commands.